samples/Samples.Server/Startup.cs

```diff
@@ -43,6 +43,9 @@ public void ConfigureServices(IServiceCollection services)
                 .AddWebSockets()
                 .AddDataLoader()
                 .AddGraphTypes(typeof(ChatSchema));
+
+            services
+                .AddSingleton<IDocumentExecuter, SubscriptionDocumentExecuter>();
```

Comment:
Not needed anymore.

Reply:
Right, following #653. Thanks.
Didn't it also resolve #588?

Reply:
Looks like it to me.  I'll close that issue.